src/client/trackers/__test__/realUserMonitoring.spec.js

```diff
@@ -53,23 +57,36 @@ describe('src/client/trackers/realUserMonitoringForPerformance', () => {
 			realUserMonitoringForPerformance();
 		});
 
-		it('creates a Perfume instance with a custom analytics tracker', () => {
-			expect(Perfume).toHaveBeenCalledTimes(1);
-			const perfumeOptions = Perfume.mock.calls[0][0];
-			expect(typeof perfumeOptions).toBe('object');
-			expect(typeof perfumeOptions.analyticsTracker).toBe('function');
+		it('listens to metrics with the web-vitals package', () => {
+			expect(onCLS).toHaveBeenCalledTimes(1);
+			expect(typeof onCLS.mock.calls[0][0]).toBe('function');
+			expect(onFCP).toHaveBeenCalledTimes(1);
+			expect(typeof onFCP.mock.calls[0][0]).toBe('function');
+			expect(onFID).toHaveBeenCalledTimes(1);
+			expect(typeof onFID.mock.calls[0][0]).toBe('function');
+			expect(onLCP).toHaveBeenCalledTimes(1);
+			expect(typeof onLCP.mock.calls[0][0]).toBe('function');
+			expect(onTTFB).toHaveBeenCalledTimes(1);
+			expect(typeof onTTFB.mock.calls[0][0]).toBe('function');
+		});
+
+		it('uses the same handler for all metrics', () => {
```

Comment:
**question:** The handler is the `recordMetric` function? Oh yes, I can see from a few lines below that this is the case 👍